tests/suites/os/tests/secureboot/index.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = {
+	title: 'secure boot tests',
+	tests: [
+		{
+			title: 'check secure boot',
+			run: async function(test) {
+				const securebootEnabled = await this.worker.executeCommandInHostOS(
+					[
+						'dmesg', '|', 'grep', '-q', '"Secure boot enabled"',
+						'&&', 'echo', 'true', '||', 'true'
+					],
+					this.link,
+				).then(output => { return output === 'true' });
```

Comment:
```suggestion
				).then(output => { return output === 'true' });
```
What purpose does the `.then` serve here? 
Did you really just want to make sure if the out put is true or not? 
That is being done in the assert below?

Reply:
In fact, this could probably just go into the suite, that might make more sense than adding a test module.

Reply:
Maybe we are working to satisfy the MVP right now and I am fine with that. But, I don't see a point for tests to be added if they can't throw any errors because who is going to be monitoring their outputs? and, if no one monitors their output. Why are they even running?

Reply:
Re-iterating @vipulgupta2048 argument. These tests are checking the golden path, that is, a system with the BIOS configured for secure boot is booting in secure boot mode and has mounted encrypted drives, and a system not configured for secure boot in BIOS is not. I think the tests should check the BIOS and then run the tests accordingly.

Reply:
I'm going to refactor this to assert that SB/FDE are enabled when supported by the firmware.